template.html

```diff
@@ -18,5 +18,6 @@
     <section id="content"></section>
 
     <footer id="footer">Footer</footer>
+    <script src="event-signup.js"></script>
```

Comment:
template.html is not going to be used its just used as a template for our own htmls.